### PR TITLE
Arxivce 1503 update arxiv docs readme

### PR DESCRIPTION
**Pull request by alisonhofer**
Updated instructions and screenshots to reflect changes in Github.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,47 +24,57 @@ group of edits or changes, please use a process like git flow. If you are not
...
 
-There click "edit" and you can edit contents of the page in github. Once you
-have made changes you can click preview to see how it will look when completed
-There are slight differences between the markdown we use and what
-github uses.
+![Screenshot of a page in Github and the location of the edit icon](/non-info/01-help-github1a.png)
 
-![gzip page](non-info/help-github2.png)
+- After you have made changes, click preview to see how it will look when completed. 
+**Please note:** There are slight differences between the markdown we use and what github uses.
```
Comment (kyokukou) [inline README.md:38]: no idea what that means

File: README.md [hunk 2]
```diff
@@ -24,47 +24,57 @@ group of edits or changes, please use a process like git flow. If you are not
...
+- Create a "new branch for this request and start a pull request" will be selected
+- Name your branch after your JIRA ticket to link it. Ex `arxivce-1503-update-arxiv-docs-readme`
+
+![Screenshot of how to create a new branch for the edits made in Github](/non-info/05-help-newBranch.png)
+
+- **Please note:** The above step will only put your changes on develop and they will not be viewable on info.arxiv.org. At this point you have only saved the edit in github.
+
+If you have more changes to make at this time you may make and commit them all on the branch you created. The additional changes you commit will be grouped together when you make your pull request in the next step.
 
 ### 2nd: Make a Pull Request from `develop` to `master`
```
Comment (kyokukou) [inline README.md:61]: theoretically they would also have to do some amount of creating a pull request from their branch to develop?

**Review by kyokukou — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (alisonhofer) [reply README.md:61]: I'm getting there. I wanted to be able to make screenshots of the next steps before I proceeded writing them.

File: README.md (hunk 1 above)
Comment (alisonhofer) [reply README.md:38]: Github uses a different type of markdown than we do with mkDocs. It uses "Github flavored" markdown to comply with Common Mark which is a markup language which has extended syntax elements making it a "superset" of markdown.

**Pull request merged**